amxmodx/meta_api.cpp

```diff
@@ -1407,8 +1407,15 @@ int	C_Cmd_Argc(void)
 // Only	here we	may	find out who is	an owner.
 void C_SetModel(edict_t *e, const char *m)
 {
-	if (e->v.owner && m[7]=='w' && m[8]=='_' && m[9]=='h')
+	if (m[9] != 'h' || m[8] != '_' || m[7] != 'w')
+	{
+		RETURN_META(MRES_IGNORED);
+	}
+
+	if (e && e->v.owner)
```

Comment:
But `SetModel` is not necessary called from the mod, a plugin (very unlikely) or a module could call directly this function and passing null pointer for some reasons. This seems more proper to let the engine crashes?